src/Process/SubdivisionControl.js

```diff
@@ -28,7 +28,7 @@ export default {
             if (node.layerUpdateState[c.id] && node.layerUpdateState[c.id].inError()) {
                 continue;
             }
-            if (c.tileInsideLimit(node, c) && !node.isColorLayerLoaded(c.id)) {
+            if (c.tileInsideLimit(node, c) && !node.isColorLayerLoaded(c)) {
```

Comment:
if `isElevationLayerLoaded` is material function, why `isColorLayerLoaded` is `tileMesh` function?

Reply:
Mainly because this PR is a part of a bigger PR. I tried to extract a meaningful part of it, but there are left over like this one that doesn't make sense.

I'm moving isColorLayerLoaded to LayeredMaterial.